tests/rkt_app_sandbox_test.go

```diff
@@ -419,6 +420,61 @@ func TestAppSandboxMount(t *testing.T) {
 	})
 }
 
+func TestAppSandboxAnnotations(t *testing.T) {
+	testSandbox(t, func(ctx *testutils.RktRunCtx, child *gexpect.ExpectSubprocess, podUUID string) {
+		for _, tt := range []struct {
+			args                    []string
+			expectedAnnotations     map[string]string
+			expectedUserAnnotations map[string]string
+		}{
+			{
+				args: []string{
+					"--annotation=foo=bar",
```

Comment:
can we use different key/value to differentiate from user annotations here?

Reply:
done